docs/reference/requests/create_transfers.md

```diff
@@ -259,11 +381,21 @@ The transfer was not created. [`Transfer.code`](../transfer.md#code) is zero, bu
 The transfer was not created. [`Transfer.debit_account_id`](../transfer.md#debit_account_id) must
 refer to an existing `Account`.
 
+This is a [`transient error`](#id_already_failed).
```

Comment:
I don't think "transient error" should be in backticks, since it isn't an identifier.